circuit_knitting_toolbox/circuit_cutting/__init__.py

```diff
@@ -23,7 +23,7 @@
 
     partition_circuit_qubits
     partition_problem
-    decompose_gates
```

Comment:
This actually needs to stay in the API docs, and I need to update the docstring to say "deprecated"